lms/djangoapps/course_blocks/transformers/tests/test_library_content.py

```diff
@@ -167,22 +168,6 @@ def test_content_library(self):
                 "Expected 'selected' equality failed in iteration {}.".format(i)
             )
 
-    def test_staff_access_to_library_content(self):
```

Comment:
Do we need to remove this, or was this just a byproduct of the cherry-pick?

Reply:
@kaizoku, it is a by-product of the cherry-pick - the test method was added to the upstream `master` branch after the Ironwood release and removed later. I left this removal in because the Ironwood test suite doesn't have this test.

Reply:
Ah gotcha, thanks. :)